src/ias_proxy/src/lib.rs

```diff
@@ -0,0 +1,37 @@
+/*
+ Copyright 2018 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+------------------------------------------------------------------------------
+*/
+
+#[macro_use]
+extern crate serde_derive;
```

Comment:
Move crate declarations to main/mod/lib

Reply:
I didn't understand, do you mean move lib.rs as a module in main?